firedrake/__init__.py

```diff
@@ -13,13 +13,17 @@
 elif not config["options"]["honour_petsc_dir"]:  # Using our own PETSC.
     os.environ["PETSC_DIR"] = os.path.join(sys.prefix, "src", "petsc")
     os.environ["PETSC_ARCH"] = "default"
-del sys, config
+del config
 
 # Ensure petsc is initialised by us before anything else gets in there.
 import firedrake.petsc as petsc
 
+# Start PETSc profiling if required (-log_view not used)
+if "-log_view" not in " ".join(sys.argv):
```

Comment:
Why not `"-log_view" not in sys.argv` ?

Reply:
Thanks for spotting this. I was originally doing some more complex regex substring stuff so this can now go.